toml/edmfx_box_advection.toml

```diff
@@ -0,0 +1,15 @@
+[C_E]
+alias = "C_E"
+value = 0.044
+type = "float"
+
+[EDMF_surface_area]
+alias = "surface_area"
+value = 0
+type = "float"
+
+[EDMF_min_area]
+alias = "min_area"
+value = 1.0e-2
```

Comment:
This seems pretty large. The result of the advection test looks a bit strange, not sure if it's related to this. We can talk more tomorrow.